src/index.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
```

Comment:
- [ ] There is an import error where the './index.css' file isn't found, removing it will resolve the problem.
![image](https://user-images.githubusercontent.com/33767951/156495910-6afa9541-85e8-4cfd-b5fd-df930c12cdd3.png)

Reply:
### Thank you very much for your feedback. I have made the changes. @omar-labana 

<img src="https://media4.giphy.com/media/RipfZWzjUDH25euMpM/giphy.gif"/>